### PR TITLE
Clarify UniFi OS config instructions and add manual update commands

### PR DESCRIPTION
**Pull request by m-d-brown**
This is based on @philsward helpful feedback in #42.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,10 +30,22 @@ You must have a Cloudflare account and your domain must be configured to point t
...
+
+To test the configuration and force an update:
+
+1. Ssh into your UniFi device
+2. Run `ps aux | grep inadyn`
+3. Note the configuration file path referenced by the running inadyn process e.g.
+   `/run/ddns-eth4-inadyn.conf`.
+4. Run `inadyn -n -1 --force -f <config-path>` e.g. `inadyn -n -1 --force -f /run/ddns-eth4-inadyn.conf`
+
+Also check `/var/log/messages` for error messages related to `inadyn`.
```
Comment (huwcbjones) [inline README.md:47]: For my USG 3P/USG Pro 4, I needed to look for errors from `ddclient` (`tail -f -n1000 /var/log/messages | grep 'ddclient'`)

**Comment by m-d-brown** [comment]
@willswire would you be able to merge? There seems to be more popularity with this project due to, I believe, the end of Google Domains

**Comment by willswire** [comment]
LGTM!

File: README.md (hunk 1 above)
Comment (m-d-brown) [reply README.md:47]: @huwcbjones want to send another PR? It's interesting that different models use different dynamic DNS clients.

**Pull request merged**